packages/core/src/attestation/Attestation.chain.ts

```diff
@@ -49,7 +50,7 @@ export async function getStoreTx(
         claimHash,
         cTypeHash,
         delegationId
-          ? { delegation: { subjectNodeId: delegationId } } // maxChecks parameter is unused on the chain side and therefore omitted
+          ? ({ delegation: { subjectNodeId: delegationId } } as any) // maxChecks parameter is unused on the chain side and therefore omitted
```

Comment:
I'll look into it

Reply:
we have `as any` here because current augmentation expects a number here; the chain will be upgraded soon to use this format but this hasn't landed yet. So anything but `any` will provoke an error here.

Reply:
Oh, okay, thanks!